cocos/asset/asset-manager/asset-manager.ts

```
@@ -113,82 +108,86 @@ export interface IAssetManagerOptions {
+     * 下载管线。
      *
      */
     public fetchPipeline: Pipeline = fetchPipeline.append(preprocess).append(fetch);
 
     /**
      * @en
-     * Url transformer
+     * Url transformer.
      *
      * @zh
-     * Url 转换器
+     * Url 转换器。
      *
      */
     public transformPipeline: Pipeline = transformPipeline.append(parse).append(replaceOverrideAsset).append(combine);
 
     /**
      * @en
-     * The collection of bundle which is already loaded, you can remove cache with [[removeBundle]]
+     * The collection of bundle which is already loaded, you can remove cache with [[removeBundle]].
      *
      * @zh
-     * 已加载 bundle 的集合， 你能通过 [[removeBundle]] 来移除缓存
+     * 已加载 bundle 的集合， 你能通过 [[removeBundle]] 来移除缓存。
      *
      */
     public bundles: ICache<Bundle> = bundles;
 
     /**
      * @en
-     * The collection of asset which is already loaded, you can remove cache with [[releaseAsset]]
+     * The collection of asset which is already loaded, you can remove cache with [[releaseAsset]].
      *
      * @zh
-     * 已加载资源的集合， 你能通过 [[releaseAsset]] 来移除缓存
+     * 已加载资源的集合， 你能通过 [[releaseAsset]] 来移除缓存。
      */
     public assets: ICache<Asset> = assets;
 
     /**
-     * @internal only using in L10N for now
+     * @engineInternal only using in L10N for now
      */
     public readonly assetsOverrideMap = assetsOverrideMap;
 
+    /**
+     * @engineInternal
+     */
     public generalImportBase = '';
+    /**
+     * @engineInternal
+     */
     public generalNativeBase = '';
 
     /**
      * @en
-     * Manage relationship between asset and its dependencies
+     * Manage relationship between asset and its dependencies.
      *
      * @zh
-     * 管理资源依赖关系
+     * 管理资源依赖关系。
      */
     public dependUtil = dependUtil;
 
     /**
      * @en
-     * Whether or not load asset forcibly, if it is true, asset will be loaded regardless of error
+     * Whether or not load asset forcibly, if it is true, asset will be loaded regardless of error.
```

A flag indicates whether to force loading assets and ignore errors.